local/new.go

```diff
@@ -64,6 +69,52 @@ func NewImage(repoName string, dockerClient DockerClient, ops ...imgutil.ImageOp
 	}, nil
 }
 
+// NewIndex will return a new local Docker ImageIndex that can be modified and saved to a registry
+func NewIndex(repoName string, ops ...Option) (idx *ImageIndex, err error) {
+	var idxOps = &imgutil.IndexOptions{}
+	for _, op := range ops {
+		if err = op(idxOps); err != nil {
+			return idx, err
+		}
+	}
+
+	if err = index.ValidateRepoName(repoName, idxOps); err != nil {
+		return idx, err
+	}
+
+	path, err := layout.FromPath(filepath.Join(idxOps.XdgPath, imgutil.MakeFileSafeName(repoName)))
+	if err != nil {
+		return idx, err
+	}
+
+	imgIdx, err := path.ImageIndex()
+	if err != nil {
+		return idx, err
+	}
+
+	mfest, err := imgIdx.IndexManifest()
+	if err != nil {
+		return idx, err
+	}
+
+	if mfest == nil {
+		return idx, imgutil.ErrManifestUndefined
+	}
+
+	if mfest.MediaType != ggcrTypes.DockerManifestList {
+		return nil, errors.New("no docker image index found")
+	}
+
+	cnbIndex, err := imgutil.NewCNBIndex(imgIdx, *idxOps)
+	if err != nil {
+		return idx, err
+	}
+
+	return &ImageIndex{
+		CNBIndex: cnbIndex,
+	}, nil
+}
+
 func defaultPlatform(dockerClient DockerClient) (imgutil.Platform, error) {
 	daemonInfo, err := dockerClient.ServerVersion(context.Background())
```

Comment:
since we have moved to GGCR, do we still need to consider using Docker Client?

Reply:
We still talk with the daemon, so yes we still need the docker client. Natalie is working on isolating the code for the interaction with the daemon and keeping all the common stuff around the v1.Image for the local, remote and layout package.